store/index.js

```diff
@@ -0,0 +1,22 @@
+export const state = () => ({
+    user: {},
+    messages: [],
+    users: []
+})
+/* Saves data */
+export const mutations = {
+    setUser(state, user) {
+        state.user = user
+    },
+    clearData(state) {
+        state.messages = [],
+            state.user = {},
+            state.users = []
```

Comment:
```suggestion
        state.messages = [];
        state.user = {};
        state.users = [];
```
![think](https://user-images.githubusercontent.com/4405297/120099697-96182100-c145-11eb-81bd-84544e58f37c.gif)